game-core/src/main/java/games/strategy/triplea/ui/AbstractForumPosterPanel.java

```diff
@@ -17,13 +17,14 @@
 
   IPlayerBridge playerBridge;
   PbemMessagePoster pbemMessagePoster;
-  ForumPosterComponent forumPosterComponent;
+  final ForumPosterComponent forumPosterComponent;
   private final JLabel actionLabel;
   private TripleAFrame tripleAFrame;
 
   AbstractForumPosterPanel(final GameData data, final MapPanel map) {
     super(data, map);
     actionLabel = new JLabel();
+    forumPosterComponent = new ForumPosterComponent(data, this::performDone, getTitle());
```

Comment:
Both child classes instatiated and set the parent 'forumPosterComponent' exactly the same way in their constructors. Consolidated this to the parent class here.